src/plugin/memory-webview-main.ts

```diff
@@ -253,22 +228,55 @@ export class MemoryWebview implements vscode.CustomReadonlyEditorProvider {
+    }
+
     protected getMemoryViewSettings(messageParticipant: WebviewIdMessageParticipant, title: string): MemoryViewSettings {
         const memoryInspectorConfiguration = vscode.workspace.getConfiguration(manifest.PACKAGE_NAME);
         const bytesPerMau = memoryInspectorConfiguration.get<number>(manifest.CONFIG_BYTES_PER_MAU, manifest.DEFAULT_BYTES_PER_MAU);
         const mausPerGroup = memoryInspectorConfiguration.get<number>(manifest.CONFIG_MAUS_PER_GROUP, manifest.DEFAULT_MAUS_PER_GROUP);
         const groupsPerRow = memoryInspectorConfiguration.get<manifest.GroupsPerRowOption>(manifest.CONFIG_GROUPS_PER_ROW, manifest.DEFAULT_GROUPS_PER_ROW);
-        const endianness = memoryInspectorConfiguration.get<Endianness>(manifest.CONFIG_ENDIANNESS, manifest.DEFAULT_ENDIANNESS);
+        const endianness = memoryInspectorConfiguration.get<manifest.Endianness>(manifest.CONFIG_ENDIANNESS, manifest.DEFAULT_ENDIANNESS);
         const scrollingBehavior = memoryInspectorConfiguration.get<ScrollingBehavior>(manifest.CONFIG_SCROLLING_BEHAVIOR, manifest.DEFAULT_SCROLLING_BEHAVIOR);
         const visibleColumns = CONFIGURABLE_COLUMNS
             .filter(column => vscode.workspace.getConfiguration(manifest.PACKAGE_NAME).get<boolean>(column, false))
             .map(columnId => columnId.replace('columns.', ''));
-        const addressPadding = AddressPaddingOptions[memoryInspectorConfiguration.get(manifest.CONFIG_ADDRESS_PADDING, manifest.DEFAULT_ADDRESS_PADDING)];
+        const addressPadding = memoryInspectorConfiguration.get(manifest.CONFIG_ADDRESS_PADDING, manifest.DEFAULT_ADDRESS_PADDING);
         const addressRadix = memoryInspectorConfiguration.get<number>(manifest.CONFIG_ADDRESS_RADIX, manifest.DEFAULT_ADDRESS_RADIX);
         const showRadixPrefix = memoryInspectorConfiguration.get<boolean>(manifest.CONFIG_SHOW_RADIX_PREFIX, manifest.DEFAULT_SHOW_RADIX_PREFIX);
+        const autoRefresh = memoryInspectorConfiguration.get<manifest.AutoRefresh>(manifest.CONFIG_AUTO_REFRESH, manifest.DEFAULT_AUTO_REFRESH);
+        const autoRefreshDelay = memoryInspectorConfiguration.get<number>(manifest.CONFIG_AUTO_REFRESH_DELAY, manifest.DEFAULT_AUTO_REFRESH_DELAY);
         return {
             messageParticipant, title, bytesPerMau, mausPerGroup, groupsPerRow,
-            endianness, scrollingBehavior, visibleColumns, addressPadding, addressRadix, showRadixPrefix
+            endianness, scrollingBehavior, visibleColumns, addressPadding, addressRadix, showRadixPrefix,
+            autoRefresh, autoRefreshDelay
+        };
+    }
+
+    protected handleSessionEvent(participant: WebviewIdMessageParticipant, event: SessionEvent): void {
+        if (isSessionEvent('active', event)) {
+            this.setSessionContext(participant, this.createContext(event.session?.raw));
+            return;
+        }
+        // we are only interested in the events of the active session
+        if (!event.session?.active) {
+            return;
+        }
```

Comment:
Is this true? Suppose that a view wanted to be updated on stopped events, and a background session stopped - would we want to ignore that? (though VSCode might automatically foreground the session...)